### ISSUE TITLE
Option for Language selection

### ISSUE THREAD

**Issue by srikanthlogic** [yuvipanda/yenWikipedia#2]
Option for language selection is required. If possible, read into user's google data to determine the list of languages and autoselect one as default.

**Comment by yuvipanda** [comment]
I don't think we can read into Google's Data here. Will add option page soon :)

**Comment by yuvipanda** [comment]
Okay, works in v0.1.9.1. Needs better UI though.

**Comment by srikanthlogic** [comment]
A dropdown box with full name is required. Not many know the ISO code properly. common urban myth : Kannada = ka where as its kn

**Comment by srikanthlogic** [comment]
Infact it must be a check box / dual list box with an add option. Just because we lack linguistic skills doesnt mean we must think others too will know onlyone other language.

**Comment by yuvipanda** [comment]
Needs better UI, as I said. Will work on that today. Open a separate issue for 'multiple language links' :)

**Comment by yuvipanda** [comment]
Consider it fixed in v0.2 :)

**Comment by srikanthlogic** [comment]
Make the options page consistant in FF port as well.

**Comment by tecoholic** [comment]
List the changes required in the FF port options page.

**Comment by srikanthlogic** [comment]
In short make FF page similar to chrome with multiselect. Add be open for more changes soon :)

**Comment by yuvipanda** [comment]
Awww, blast from the past! <3

**Comment by srikanthlogic** [comment]
👋 

You could use it for "How it started, How its going" meme.
